models/swapr/ethereum/swapr_ethereum_trades.sql

```diff
@@ -0,0 +1,107 @@
+{{ config(
+    alias ='trades',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'trace_address'],
+    post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "project",
+                                    "swapr",
+                                    \'["cryptoleishen"]\') }}'
+    )
+}}
+
+{% set project_start_date = '2020-12-09' %}
+
+WITH dexs AS
+(
+    -- Swapr
+    SELECT
+        t.evt_block_time AS block_time,
+        t.`to` AS taker,
+        '' AS maker,
+        CASE WHEN amount0Out = 0 THEN amount1Out ELSE amount0Out END AS token_bought_amount_raw,
```

Comment:
just a comment for sharing past experiences: on the decoded tables, columns such as ```amount0out``` or ```amount1out``` are string, which has led to some edge use cases where equality operators don't compute properly without single quotes around the value, i.e. ```'0'``` but i think when it's just ```=``` then it's fine. usually the ```>``` or ```<``` are the culprits

Reply:
thank you for reminding me @jeff-dude